Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -2079,25 +2081,29 @@ void IMixedRealityPointerHandler.OnPointerUp(MixedRealityPointerEventData eventD
                 }
 
                 //Release the pointer
-                currentPointer.IsTargetPositionLockedOnFocusLock = true;
- 
-                 ResetState();
+                currentPointer.IsTargetPositionLockedOnFocusLock = oldIsTargetPositionLockedOnFocusLock;
+
+                ResetState();
             }
         }
 
         ///<inheritdoc/>
         void IMixedRealityPointerHandler.OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            currentPointer = eventData.Pointer;
+            oldIsTargetPositionLockedOnFocusLock = currentPointer.IsTargetPositionLockedOnFocusLock;
+
             //Quick check for the global listener to bail if the object is not in the list
             if (eventData.Pointer?.Result?.CurrentPointerTarget == null
                 || !ContainsNode(eventData.Pointer.Result.CurrentPointerTarget.transform) || initialFocusedObject != null)
             {
                 return;
             }
 
-            currentPointer = eventData.Pointer;
-
-            currentPointer.IsTargetPositionLockedOnFocusLock = false;
+            if (!(eventData.Pointer is IMixedRealityNearPointer) && eventData.Pointer.Controller.IsRotationAvailable)
```

Comment:
Since this is cached at the beginning of the method now:

```suggestion
            if (!(currentPointer is IMixedRealityNearPointer) && currentPointer.Controller.IsRotationAvailable)
```